images/00-base/Dockerfile

```diff
@@ -37,7 +37,6 @@ RUN apk --no-cache add \
     procps \
     qemu-guest-agent \
     rng-tools \
-    rng-tools-openrc \
```

Comment:
I guess this was included by mistake? :D

Reply:
nah, as per discussion with you a number of weeks ago it wasn't necessary and so i threw it in with this change

Reply:
... because it gets installed implicitly

Reply:
Actually, I remember that discussion ending differently.

rng-tools-openrc wasn't added implicilty in 3.9 however haveged-openrc was added implicitly by something else that you weren't able to track down properly.

Reply:
Strike that. Just did an `apk add rng-tools` in a alpine 3.9 container and it installed rng-tools-openrc too.